SOURCES.rst

```diff
@@ -43,3 +43,5 @@
 +----------------+------------------------------------------------------------------------------------------------------+----------------------------------------------------+
 |postgresql      | https://www.postgresql.org/support/security/                                                         |postgresql                                          |
 +----------------+------------------------------------------------------------------------------------------------------+----------------------------------------------------+
+|elixir_security | https://github.com/dependabot/elixir-security-advisories'                                            |hex packages                                        |
```

Comment:
Remove the single quote at end of the url

Reply:
Oh Sorry I missed it.I will make the change